homework1_portfolio_styles.css

```diff
@@ -0,0 +1,75 @@
+body {
+	font-family: helvetica;
+}
+
+nav {
+	font-size: 28px;
+	padding-top: 10px;
+	text-align: center;
+
+}
+
+h1 {
+	font-size: 40px;
+	text-align: center;
+}
+
+h2 {
+	font-size: 26px;
+	padding-left: 100px;
+}
+
+h4 {
+	font-size: 18px;
+	padding-left: 100px;
+}
+
+img { 
+	float: left;
```

Comment:
I love that you are experimenting with float here!  I did see that the image is sitting below the paragraphs on the page - is it supposed to be to the left of the paragraphs (so the image and paragraph are side-by-side)?  If so, try swapping the order of the ```<img>``` and ```<p>``` elements in your HTML.  (Or if not, try it and see what happens! :))